cmd/autoheal/healer.go

```diff
@@ -200,6 +193,44 @@ func (h *Healer) Run(stopCh <-chan struct{}) error {
 	return nil
 }
 
+// Reload config files and build config and cache.
+//
+func (h *Healer) ReloadConfig() {
+	// Reload config files
+	glog.Info("Config files reload request: reloding files.")
+	h.loader.Load()
+
+	// Reload rules qeue
+	h.resetRulesQueue()
+}
+
+// Reset the rules queue, and reload from config.
+//
+func (h *Healer) resetRulesQueue() {
+	// Delete all items in qeue
```

Comment:
The rules worker may need to do something when deleting a rule: closing certain connections to AWX, updating some metrics, etc. So I think it is better if you do this creating a new `RuleChange` for each rule, and letting the rule worker do the actual work.

Reply:
Thanks, will look into it :+1: